### PR TITLE
Added committee overview tab

### PR DESCRIPTION
**Pull request by phgross**
Added first version of the overview tab, for the committee Tabbed view.
bildschirmfoto 2015-03-02 um 15 52 04

@deiferni please have a look ...

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,6 +5,12 @@ Changelog
 4.2.0 (unreleased)
 ------------------
 
+- Added committe overview tab.
```
Comment (lukasgraf) [inline docs/HISTORY.txt]: `committe` -> `committee`

File: opengever/meeting/browser/committee.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+from five import grok
+from opengever.meeting import _
+from opengever.meeting.committee import ICommittee
+from opengever.tabbedview.browser.base import OpengeverTab
+
+
+class CommitteeOverview(grok.View, OpengeverTab):
+    """The overview tab for the commite tabbeview.
```
Comment (lukasgraf) [inline opengever/meeting/browser/committee.py]: `committe` -> `committee`

File: opengever/meeting/browser/committee.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    grok.context(ICommittee)
+    grok.name('tabbedview_view-overview')
+    grok.require('zope2.View')
+    grok.template('overview')
+
+    show_searchform = False
+
+    def boxes(self):
+        items = [
+            [{'id': 'coming_meetings',
```
Comment (lukasgraf) [inline opengever/meeting/browser/committee.py]: `coming` -> `upcoming`

File: opengever/meeting/browser/committee.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+              'label': _('label_coming_meetings', default=u'Coming meetings'),
```
Comment (lukasgraf) [inline opengever/meeting/browser/committee.py]: Same here, `*coming*` -> `*upcoming*`

File: opengever/meeting/browser/committee_templates/overview.pt [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+      <h2 i18n:translate="" tal:content="box/label|box/id"></h2>
+
+      <ul tal:condition="box/content">
+        <li tal:repeat="item box/content">
+          <span tal:replace="structure item"></span>
+        </li>
+
+        <li class="moreLink" tal:condition="box/href" >
+          <a tal:attributes="href box/href" i18n:translate="">See more</a>
```
Comment (lukasgraf) [inline opengever/meeting/browser/committee_templates/overview.pt]: `See more` -> `Show more`

File: opengever/meeting/tests/test_committee_overview.py [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+                       date_to=date.today() + timedelta(days=1)))
+
+        browser.login().open(self.committee, view='tabbedview_view-overview')
+
+        self.assertEquals(
+            'http://nohost/plone/committee-1/member/1',
+            browser.css('#current_membersBox li a').first.get('href'))
+
+    @browsing
+    def test_comming_meetings_box_only_lists_meetings_in_the_future(self, browser):
```
Comment (lukasgraf) [inline opengever/meeting/tests/test_committee_overview.py]: Also here -> `upcoming`

File: opengever/meeting/upgrades/configure.zcml [hunk 7]
```diff
@@ -120,4 +120,14 @@
         directory="profiles/4211"
         />
 
+    <!-- 4211 -> 4212 -->
+    <genericsetup:upgradeStep
+        title="Add overview tab action for the commite tabbedview."
```
Comment (lukasgraf) [inline opengever/meeting/upgrades/configure.zcml]: `commite` -> `commitee`

**Comment by lukasgraf** [comment]
Besides the spelling issues I commented on: :+1: !

**Comment by deiferni** [comment]
:+1:

**Pull request merged**